src/TurnByTurn/RoutingSample.Desktop/MainWindow.xaml

```diff
@@ -44,8 +44,7 @@
         <esri:MapView x:Name="MapView"
                       Map="{Binding Map}"
                       GraphicsOverlays="{Binding RouteGraphicsOverlays}"
-                      Grid.Row="2"
-                      GeoViewTapped="MapView_GeoViewTapped">
```

Comment:
Is there a reason why you removed the GeoViewTapped event?  The suer will not be able to click on the map and choose a location to use.